src/Types.jl

```diff
@@ -372,6 +372,7 @@ function stdlibs()
     return STDLIB[]
 end
 is_stdlib(uuid::UUID) = uuid in keys(stdlibs())
+is_stdlib(name::String) = name in values(stdlibs())
```

Comment:
This seems bad for when we have packages bwith same names as stdlibs.
Like TOML.jl?

Reply:
Yea this shouldn't be necessary since we know the UUIDs of everything.

Reply:
But we don't directly know the uuids of deps of deps in the manifest. That's a string name list. Perhaps the proper thing is to convert those to PkgId lists, but I thought that if this was ok, it would be faster. 

Either way, you have to match a string name value. This way just skips the uuid comparison.

Reply:
Perhaps it shouldn't be a separate function though.. I could internalize it, to prevent promoting use.

Reply:
Now a local method 👍🏻

Reply:
> But we don't directly know the uuids of deps of deps in the manifest.

Yes we do? Everything is there.

Reply:
I think I'm being slow.. How do we tell the difference between a dep in the manifest entries that's a stdlib vs. a same-named package? The deps list is a list of names, not uuids
```
[[AbstractAlgebra]]
deps = ["InteractiveUtils", "LinearAlgebra", "Markdown", "Random", "SparseArrays", "Test"]
git-tree-sha1 = "8fa03ecf25341ff3e8fb301dba3f41c6fe09952e"
uuid = "c3fe647b-3220-5bb0-a1ea-a7954cac585d"
version = "0.10.0"
```

Reply:
Those names are required to be keys within the `Manifest`; so if someone else had a `deps = ["AbstractAlgebra"]` you'd look at the entry you see above you right there and pull out the `uuid`.  A `Manifest` can only have a single mapping for each name, so there's a direct correspondence.

That being said, I think this is a mostly academic distinction, as for stdlibs with packages that have the same name, I'm pretty sure we're going to ensure they have the same UUID as well, so that earlier versions of Julia can transparently deal with newer packages in an intelligent way.

Reply:
> A Manifest can only have a single mapping for each name, so there's a direct correspondence.

No, there can be multiple mappings. It is just in the common case where all names are unique that the deps are a list like that, otherwise deps are listed as `name = uuid`. E.g.
```
[[JSON]]
git-tree-sha1 = "81690084b6198a2e1da36fcfda16eeca9f9f24e4"
uuid = "682c06a0-de6a-54ab-a142-c8b1cf79cde6"
version = "0.21.1"

    [JSON.deps]
    Dates = "ade2ca70-3891-5945-98fb-dc099432e06a"
    Mmap = "a63ad114-7e13-5084-954f-fe012c677804"
    Parsers = "69de0a69-1ddd-5017-9359-2bf0b02dc9f0"
    Unicode = "4ec0a83e-493e-50e2-b9ac-8f72acf5a8f5"

[[Parsers]]
path = "Parsers"
uuid = "27091c14-f214-46fa-a0da-49c36ffec0b3"
version = "0.1.0"
[[Parsers]]
deps = ["Dates", "Test"]
git-tree-sha1 = "8077624b3c450b15c087944363606a6ba12f925e"
uuid = "69de0a69-1ddd-5017-9359-2bf0b02dc9f0"
version = "1.0.10"
```

Reply:
Ok, I get why what I'm doing is wrong, but not sure how to fix it. The keys in the result of `Pkg.Types.read_manifest(ctx.env.manifest_file)` are the UUIDs. I don't see an efficient way to convert the deps list into a list of UUIDs.

Reply:
I got it now.. Commit incoming

Reply:
Ah, thanks for that Fredrik, I didn't know that, or forgot that I ever knew it, I guess.  :P